src/plugins/controls/public/range_slider/components/range_slider_button.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { EuiFieldNumber, EuiFormControlLayoutDelimited } from '@elastic/eui';
+
+import { useRangeSlider } from '../embeddable/range_slider_embeddable';
+
+import './range_slider.scss';
+
+export const RangeSliderButton = ({
+  onClick,
+}: {
+  onClick: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+}) => {
+  const rangeSlider = useRangeSlider();
+
+  const min = rangeSlider.select((state) => state.componentState.min);
+  const max = rangeSlider.select((state) => state.componentState.max);
+  const isInvalid = rangeSlider.select((state) => state.componentState.isInvalid);
+
+  const id = rangeSlider.select((state) => state.explicitInput.id);
+  const value = rangeSlider.select((state) => state.explicitInput.value) ?? ['', ''];
+
+  const isLoading = rangeSlider.select((state) => state.output.loading);
+
+  return (
+    <EuiFormControlLayoutDelimited
+      fullWidth
+      isLoading={isLoading}
+      className="rangeSliderAnchor__button"
+      data-test-subj={`range-slider-control-${id}`}
+      onClick={onClick}
+      startControl={
+        <EuiFieldNumber
+          controlOnly
+          fullWidth
+          value={value[0] === String(min) ? '' : value[0]}
```

Comment:
This makes it so that, if you **manually** select the min/max values, they are treated as "no selection" - this did not happen before:

| Before | After |
|--------|-------|
| ![Jun-09-2023 09-44-03](https://github.com/elastic/kibana/assets/8698078/4eb38fe4-9157-4071-bb0f-fbd29ce459c5) | ![Jun-09-2023 09-43-14](https://github.com/elastic/kibana/assets/8698078/0cea71ed-ca58-4ace-8c52-4f0ec9390ae2) |